src/dataviews/histogram-dataview/histogram-data-model.js

```diff
@@ -1,4 +1,5 @@
 var _ = require('underscore');
+var BackboneCancelSync = require('../../util/backbone-abort-sync');
```

Comment:
and `BackboneCancelSync` in [here](https://github.com/CartoDB/cartodb.js/blob/v4/src/dataviews/dataview-model-base.js#L3)

Which one should we adopt?